services/graphql/src/resolvers/config.js

```diff
@@ -3,6 +3,7 @@ const {
   CONTACT_URL,
   CONTACT_TEXT,
   LEADERSHIP_ENABLED,
+  LEADERSHIP_COMPANY_LABEL,
```

Comment:
This new variable needs to be defined in the envalid configuration using the `str` validator. When doing so, please confirm the behavior when the label isn't defined -- it should be nulled out in the graphql response when not present.

Reply:
@solocommand wouldn't this return an empty string if it was setup in the env.js as a `str` or am i missing something?